packages/styled-components/src/models/ServerStyleSheet.js

```diff
@@ -1,109 +1,107 @@
 // @flow
 /* eslint-disable no-underscore-dangle */
+
 import React from 'react';
-import stream from 'stream';
 
-import { IS_BROWSER, SC_STREAM_ATTR } from '../constants';
+import {
+  IS_BROWSER,
+  SC_ATTR,
+  SC_VERSION_ATTR,
+  SC_VERSION
+} from '../constants';
+
 import StyledError from '../utils/error';
-import StyleSheet from './StyleSheet';
+import getNonce from '../utils/nonce';
+import StyleSheet from '../sheet';
 import StyleSheetManager from './StyleSheetManager';
 
 declare var __SERVER__: boolean;
 
 const CLOSING_TAG_R = /^\s*<\/[a-z]/i;
 
 export default class ServerStyleSheet {
-  instance: StyleSheet;
-
-  masterSheet: StyleSheet;
+  sheet: StyleSheet;
 
   sealed: boolean;
 
   constructor() {
-    /* The master sheet might be reset, so keep a reference here */
-    this.masterSheet = StyleSheet.master;
-    this.instance = this.masterSheet.clone();
+    this.sheet = new StyleSheet(true);
     this.sealed = false;
   }
 
-  /**
-   * Mark the ServerStyleSheet as being fully emitted and manually GC it from the
-   * StyleSheet singleton.
-   */
-  seal() {
```

Comment:
If we remove this how does someone externally seal in a streaming error scenario?

Reply:
It's temporarily removed, but I might put it back later :+1:

Reply:
Yeah we need this!

Reply:
So yea, we need it for the errors on streaming when a sheet is used twice but that's it 😅